File: l1_intro/table.cpp
#include <iostream>
using namespace std;

int main(int args, char **argv)
{
  int number = 0;
  cout << "Enter the number for table ";
  cin >> number;

  int limit = 0;
  cout << "upto ";
  cin >> limit;

  cout << "The table is" << endl;

  for (int i = 1; i <= limit; i++)
  {
    cout << number << " * " << i << " = " << i * number << endl;
  }

  return 0;
}

File: l1_intro/Table.java
import java.util.Scanner;
class Table{
  public static Scanner scn=new Scanner(System.in);
  public static void main(String args[]){
    System.out.print("Enter the number for table  ");
    int number=scn.nextInt();
    System.out.print("Enter limit ");
    int limit=scn.nextInt();

    for(int i=1;i<=limit;i++){
      System.out.println(number+" * "+i+" = "+number*i);
    }
  }
}